### ISSUE TITLE
🐞🐋 unable to understand docker compose up logs - SELKS not starting

### ISSUE THREAD

**Issue by Balukn11** [StamusNetworks/SELKS#436]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

[error.txt](https://github.com/StamusNetworks/SELKS/files/11119721/error.txt)

### Expected Behavior

Should be able to view the SELKS website in my localhost

### Docker version

Docker version 23.0.2, build 569dd73

### Docker version

Docker version 23.0.2, build 569dd73

### OS Version

Ubuntu 22.04.2 LTS

### Content of the environnement File

COMPOSE_PROJECT_NAME=SELKS
INTERFACES= -i eth0 -i enP28509s1
SCIRIUS_DEBUG=True
NGINX_EXEC=nginx-debug
ELASTIC_DATAPATH=/mnt/home/
SCIRIUS_SECRET_KEY=ANIwz8fPrn5WcA4G8CDSCtX_IdRBtIhaOXd2rbdVwxg
PWD=$\{PWD\}

### Version of SELKS

commit d57826018298590f484b5ec2e31e6fe55c319be1 (HEAD -> master, origin/master, origin/HEAD)
Author: Peter Manev <pmanev@stamus-networks.com>
Date:   Sun Mar 5 11:15:23 2023 +0100

    docker: Add fixes for docker compose v2

**Comment by pevma** [comment]
Did the basic install finish ok ? https://github.com/StamusNetworks/SELKS/wiki/Docker#basic-installation  
It seems like the Scirius container did not start properly - can you make it down then up please?  
Also just a sanity check - you have at latest 2CPU and 10GB RAM dedicated for the containers?

**Comment by Balukn11** [comment]
Yeah, the basic installation completed properly and all the containers started successfully, you can just check it from the logs. I hope the sanitary checks are satisfied, because I'm using a virtual machine with 64GB RAM
